RDAPI-23024 fix yaml pk flaw revisit doc for obsolete notions (#1722)
Axway/axway-open-docs: content/en/docs/apim_yamles/yamles_structure.md

name: Filter Database IP
```

It now has a YamlPK of `/Policies/App Policies/Core/Filter Database IP`. This means that all other entities pointing to this policy through a reference field must be changed to reflect this.

> If you are using the `EntityStore` API, then all of that is handled internally and referring entities will be updated.

Reviewer: ```suggestion
If you are using the `EntityStore` API, in case an entity's YamlPK is altered due to a change in any of its key fields, all referring entities are automatically updated to reflect the change.
 
```

**Two or more entities with the same key fields at the same level in the hierarchy**:

> In that case the YamlPK has is a little bit different.

Reviewer: ```suggestion
In this case the YamlPK is formed differently to avoid ambiguity.
```

For instance if you have:

* `/Policies/App Policies/Core`: a policy

> * `/Policies/App Policies/Core`: (container other policy)

Reviewer: (container other policy) => (container of other policies) or (contain other policies)

* `/Policies/App Policies/Core/Throttling`: a policy

> There a conflict, YAML Entity store handle by setting their respective YamlPK to:

Reviewer: ```suggestion
There is a conflict, the YAML Entity Store handles this by adding the parent container to the respective YamlPK:
```

Author: It's the type that we add but kept it as you suggested.

Reviewer: Maybe this would be more accurate then ? => There is a conflict, the YAML Entity Store handles this by adding the types to the respective YamlPK:

Reviewer: Already done
